### PR TITLE
updated concepts

### PR DESCRIPTION
**Pull request by sriumcp**
Updated concepts that includes descriptions of traffic controller and ABn SDK.

### CODE REVIEW

File: docs/images/iter8usecases.png
Comment (kalantar) [inline docs/images/iter8usecases.png:1]: We have this image twice. In `docs/getting-started/images/iter8usecases.png` and `docs/images/iter8usecases.png`. Can we just have once instance of the image?

File: docs/getting-started/images/iter8usecases.png
Comment (kalantar) [inline docs/getting-started/images/iter8usecases.png:1]: We have this image twice. In `docs/getting-started/images/iter8usecases.png` and `docs/images/iter8usecases.png`. Can we just have once instance of the image?

File: docs/getting-started/concepts.md [hunk 1]
```diff
@@ -3,47 +3,52 @@ template: main.html
...
 
-![Iter8 experiment](../images/underthecovers.png)
+    With Iter8, the barrier to entry for end-users is significantly reduced. In particular, by just providing names of their ML serving resources, and (optional) traffic weights/labels, endusers can get started with their release optimization use cases rapidly. Further, Iter8 does not limit the capabilities of the underlying service mesh in anyway. This means more advanced teams still get to use all the power of the service-mesh alongside the reliability and ease-of-use that Iter8 brings.
```
Comment (kalantar) [inline docs/getting-started/concepts.md:37]: "endusers" should be two words

File: docs/getting-started/concepts.md [hunk 2]
```diff
@@ -3,47 +3,52 @@ template: main.html
...
+
+=== "Client SDK"
+
+    Iter8 provides a client-side SDK to facilitate routing as well as metrics collection task associated with distributed (i.e., client-server architecture-based) A/B/n testing in Kubernetes. The following picture illustrates the use of the SDK for A/B testing.
+
+    ![A/B testing](../tutorials/abn/images/abn.png)
+
+    Iter8's SDK is designed to handle user stickiness, collection of business business, and decoupling of front-end and back-end releases processes during A/B/n testing.
```
Comment (kalantar) [inline docs/getting-started/concepts.md:45]: "business business" should be "business metrics" ?

**Review by kalantar — CHANGES_REQUESTED** [review]
minor issues

File: docs/getting-started/concepts.md [hunk 3]
```diff
@@ -3,47 +3,52 @@ template: main.html
...
 
-In addition to predefined tasks, Iter8 packs a number of powerful features that facilitate experimentation. They include the following.
+    ![Iter8 experiment](images/kubernetesusage.png)
 
-1.  **HTML/text reports** that promote end-user understanding of experiment results through visual insights.
-2.  **Assertions** that verify whether the target app satisfies the specified SLOs or not during/after an experiment.
-3.  **Multi-loop experiments** that can be executed periodically instead of just once (single-loop).
+    In addition to performance testing and SLO validation for HTTP and gRPC services, Iter8 experiments can also be used to compare versions of an app/ML inference service in terms of their business metrics, and performing SLO validation using metrics from custom databases (such as Prometheus).
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:27]:
```suggestion
    In addition to performance testing and SLO validation for HTTP and gRPC services, Iter8 experiments can also be used to compare versions of an app/ML inference service in terms of their business metrics, and perform SLO validation using metrics from custom databases (such as Prometheus).
```

File: docs/getting-started/concepts.md [hunk 4]
```diff
@@ -3,47 +3,52 @@ template: main.html
...
+    ![Blue-Green](../tutorials/integrations/kserve-mm/images/blue-green.png)
+    
+    As part of the dynamic reconfiguration of route resources, the Iter8 controller also looks into readiness (for e.g., in a KServe modelmesh based inference service), availability (for e.g., in a Kubernetes deployment based inference service) and other relevant status conditions before configuring traffic splits to candidate versions. Similarly, before candidate versions are deleted, the Iter8 controller uses finalizers to first ensure that all traffic flows to the primary version of the inference service. This makes for a very high-degree of reliability and zero-downtime/loss-less rollouts of new app/inference service versions. Users do not get this level of reliability out-of-the-box with a vanilla service mesh.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:35]:
```suggestion
    As part of the dynamic reconfiguration of route resources, the Iter8 controller also looks into readiness (for e.g., in a KServe modelmesh-based inference service), availability (for e.g., in a Kubernetes deployment-based inference service) and other relevant status conditions before configuring traffic splits to candidate versions. Similarly, before candidate versions are deleted, the Iter8 controller uses finalizers to first ensure that all traffic flows to the primary version of the inference service. This makes for a very high-degree of reliability and zero-downtime/loss-less rollouts of new app/inference service versions. Users do not get this level of reliability out-of-the-box with a vanilla service mesh.
```

**Review by Alan-Cha — DISMISSED** [review]

File: docs/getting-started/concepts.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ template: main.html
 ---
 
 # Iter8
-Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data science teams. Iter8 detects new versions of Kubernetes apps and ML inference services in the cluster, and validates that they perform well and maximize business value.
+Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data science teams. Iter8 automates traffic control for new versions of apps/ML inferences services in the cluster, and automates validation of their performance and business metrics.
```
Comment (Alan-Cha) [inline docs/getting-started/concepts.md:6]: "ML inferences services" sounds strange to me

**Review by Alan-Cha — APPROVED** [review]

File: docs/getting-started/concepts.md (hunk 5 above)
Comment (sriumcp) [reply docs/getting-started/concepts.md]: Fixed.

**Review by kalantar — APPROVED** [review]
lgtm

**Pull request merged**